src/screens/SendFunds/04-Summary.js

```diff
@@ -159,6 +163,35 @@ class SendSummary extends Component<
     }
   };
 
+  nonceMaxAmount = 0;
+  syncMaxAmount = async () => {
+    const { account, parentAccount, navigation } = this.props;
+    if (!account) return;
+    const bridge = getAccountBridge(account, parentAccount);
+    const mainAccount = getMainAccount(account, parentAccount);
+    const transaction = navigation.getParam("transaction");
+    const nonce = ++this.nonceMaxAmount;
+
+    const useAllAmount = bridge.getTransactionExtra(
+      mainAccount,
+      transaction,
+      "useAllAmount",
+    );
+
+    if (nonce !== this.nonceMaxAmount) return;
+
+    let maxAmount;
+    if (useAllAmount)
+      maxAmount = await bridge.getMaxAmount(mainAccount, transaction);
+
```

Comment:
a `if (nonce !== this.nonceMaxAmount) return;` needs to be here (typically after every await)

Reply:
got it